Testing/SystemTests/tests/framework/ISIS_PowderPolarisTest.py

```diff
@@ -447,6 +447,21 @@ def validate(self):
         # to be updated very frequently. In the meantime, the expected peak in the PDF at ~3.9 Angstrom will be checked.
         idx = get_bin_number_at_given_r(self.pdf_output.dataX(0), 3.9)
         self.assertAlmostEqual(self.pdf_output.dataY(0)[idx], 2.8955, places=3)
+        self.assertEqual(self.pdf_output.name(), "98533_pdf_g(r)")
+
+
+class TotalScatteringPdfNameTest(systemtesting.MantidSystemTest):
+    pdf_output = None
+
+    def runTest(self):
+        setup_mantid_paths()
+        # Load Focused ws
+        mantid.LoadNexus(Filename=total_scattering_input_file, OutputWorkspace="98533-ResultTOF")
+        q_lims = np.array([2.5, 3, 4, 6, 7, 3.5, 5, 7, 11, 40]).reshape((2, 5))
+        self.pdf_output = run_total_scattering("98533", True, q_lims=q_lims, pdf_type="g(r)", pdf_output_name="test_pdf_output")
+
+    def validate(self):
+        self.assertEqual(self.pdf_output.name(), "test_pdf_output")
```

Comment:
Do we want to add a test for the default name as well or do you think that's overkill?

Reply:
Hmm, it is a good idea, but I did that implicitly by adding line 450 in the other test. Do you think it is better to separate it into its own unit test?

Reply:
Ah yes I see, sorry about that! Looks like both cases are covered so I'm happy!